CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- Added the `eprinttype` field to the General entry editor tab for users still using the default configuration. (#14555)
```

Comment:
This is an issue number, but not a proper link.

Please check the line below in the file how to create a proper link.